test/e2e/runner/load.go

```diff
@@ -21,9 +21,9 @@ func Load(ctx context.Context, testnet *e2e.Testnet) error {
 	// CPU. This gives high-throughput small networks and low-throughput large ones.
 	// This also limits the number of TCP connections, since each worker has
 	// a connection to all nodes.
-	concurrency := 64 / len(testnet.Nodes)
-	if concurrency == 0 {
-		concurrency = 1
+	concurrency := len(testnet.Nodes) * 8
+	if concurrency > 64 {
```

Comment:
Alternative suggestion:

```go
if cpu := runtime.NumCPU(); concurrency > cpu {
   concurrency = 64
}
```

Or if this is counting in goroutines instead of tasks, maybe some small multiplier on that. YMMV.

Reply:
the load generation isn't really CPU intensive, and I think it makes sense to have it be in relationship to the number of nodes, generally.